Review of general and basic usage sections

jorikvankemenade opened the pull request:
I have looked at the "General" and "Basic Usage" sections of the SDR docs. The majority of the fixes target small typos and or attempt to simplify sentences. I have also tried to introduce a more active and less descriptive voice in the docs.

Some notable changes:

- I have renamed the basic usage page (not section) to "Navigating the data repository"
- I have removed nesting from several pages. For example, I have flattened the best practices.
- I have updated the download instructions to include installation and usage of the script.

jorikvankemenade committed:
Update ignore of the build directory

In vscode the build directory is called _build. By adding a

jorikvankemenade committed:
Fix small errors in the about section

jorikvankemenade committed:
Fix layout and links in general/access

jorikvankemenade committed:
Change small typos in the best practices section

jorikvankemenade committed:
Fix typos in basic usage and change structure

I think it makes sense if we cover how to find deposits and collections
before explaning the deposit and collection landing pages. I have also
changed the page title to "navigating the data repository" to
diffirentiate from the "basic usage" section of the docs. I think this
is more clear for the prospective users.

jorikvankemenade committed:
Fix link to docs in the README

The http address times out, we need to link to the https.

jorikvankemenade committed:
Improve the how to get access section

Add working links to the helpdesk page and suggest some first actions.

jorikvankemenade committed:
Restructure the access and general info pages

The same kind off info was on different pages. I have changed the
structure to have access on one page and service information on a
different page.

jorikvankemenade committed:
Update deposit documentation

Make some textual improvements and fix the alignment of the different
steps

jorikvankemenade committed:
Fix small typos and grammar in metadata

jorikvankemenade committed:
Update and restructure download instructions

Make some textual improvements. The most significant change is the
restructring of the download script instruction. I have changed this to
a step-by-step instruction to guide the user.

jorikvankemenade committed:
Simplify about structure

jorikvankemenade committed:
Simplify structure for access page

Remove the page structure for small pages, the headings are already
clear.

jorikvankemenade committed:
Simplify content table on the best practices page

jorikvankemenade committed:
Improve access links and headings

jorikvankemenade committed:
Improve tone and style for basic usage section

jorikvankemenade committed:
Improve tone and style of the downloads section

jorikvankemenade committed:
Fix toc for the metadata page

jorikvankemenade commented on source/pages/general/about.rst:
```diff
@@ -1,40 +1,23 @@
+The repository provides access to mostly open access data which can be found and downloaded by anyone interested. You can find the data repository at https://repository.surfsara.nl. For publishing datasets an account is required, please see :ref:`this page <get-access>` for information on how to obtain an account.
 
-The Data Repository service is suitable for publishing research data especially in the field of Big Data sciences. The repository imposes no limits on the data volume and number of files, but does have some guidelines on how to efficiently store your data for publication.
+There is a direct connection between the data repository service and other SURF services such as the Data Archive. All data in the data repository will eventually end up in the data archive for long-term storage and preservation. The repository is tape-aware, this means that data can automatically be put offline when it is not required for a certain amount of time.
```
To me, the usage of tape for the SDR is more of an implementation detail. It is not necessarily important information for a user to understand the service or decide if they want to use the service. Perhaps we could just say something along the lines that the SURF data archive is used to ensure the long-term preservation of published datasets?

hevp replied:
It is important, because it might affect researcher's (automatic) workflows. These need to take into account that data can be offline when using the repository for downloads.

jorikvankemenade commented on source/pages/practices/best_practices.rst:
```diff
@@ -33,51 +22,47 @@ If you think your file format should be accepted by the repository, but is curre
 - **Maximum file size**: 4 GB using the online deposit form
 
-The maximum deposit data volume is 10 GB and the maximum number of files that can be uploaded via the onlone deposite is 10 files. If you want to deposit larger volumes of data or larger number of files in a single deposit, please contact us.
+The maximum deposit data volume is 10 GB and the maximum number of files that can be uploaded via the online deposit form is 10 files. If you want to deposit larger volumes of data or a larger number of files in a single deposit, please contact us.
```
Does this maximum also apply when uploading via the REST API or only via the form?

hevp replied:
Yes, the maximum is currently enforced when using the REST API.

jorikvankemenade commented on source/pages/practices/best_practices.rst:
```diff
@@ -33,51 +22,47 @@ If you think your file format should be accepted by the repository, but is curre
+- **Administrative metadata** provides information on user rights and management of digital objects and inlcudes elements such as license, rights management, and access control.
 
-The repository will only ask for descriptive metadata during the deposit workflow, but it is highly recommended to add links to additional documentation or files containing metadata of any kind. Any other kinds of metadata are automatically added, though not always visible for the end-user.
+The repository only requires descriptive metadata during the deposit workflow. However, it is highly recommended to add links to additional documentation or files containing metadata of any kind. Any other kinds of metadata are automatically added, though not always visible for the end-user.
```
> Any other kinds of metadata are automatically added, though not always visible for the end-user.

This sentence is not entirely clear to me, what are we trying to say?

hevp replied:
Some descriptive fields are automatically added by SDR and do not need to be added by the user.

jorikvankemenade commented:
@hevp please do not forget to check and merge this :).

hevp commented on .gitignore:
```diff
@@ -1,3 +1,3 @@
-build/
+*build/
```
Why the wildcard? It might affect deeper directories that have similar names.

jorikvankemenade replied:
Because in my local dev environment `source/_build` was created. This change would cover all possible variations and given the goal of the documentation a `*build` directory that is part of the source seemed unlikely. But I am happy to make the `.gitignore` more explicit.

hevp commented on source/pages/basics/basic_usage.rst:
```diff
@@ -1,52 +1,59 @@
-Basic Usage
-***********
+******************************
+Navigating the data repository
```
Preferably when using the term 'data repository' use title capitals.

hevp replied:
So, in general refer to the 'service' or 'repository', but when using 'data repository' as a name use title capitals.

hevp commented on source/pages/basics/basic_usage.rst:
```diff
@@ -1,52 +1,59 @@
+   **Community** is a group of common users that are part of a research domain or initiative
 
-    See :ref::`Objects <advanced-objects` for more information on object types.
+   See the :ref:`objects reference <advanced-objects>` for more information on object types.
```
Please do use the title of the page here.

hevp commented on source/pages/basics/basic_usage.rst:
```diff
@@ -1,52 +1,59 @@
+The search bar can be used to quickly find deposits and collections. It supports faceted search by prepending a field before your search term, for example *title:data*.
 
-On the buttom left column, the are  shortcuts for defining new collections, metadata schemas and groups, explained more in the :ref:`Advanced usage <advanced-usage>`.
+Clicking on the avatar in the top right corner presents you with a dropdown menu where you can locate your account details. The ``My account`` button sends you to your `account page`_. Here you can create and manage your deposits, collections, groups and, if allowed, communities and schemas. Please refer to the :ref:`advanced usage <advanced-usage>` section of the documentation for more informati ...
```
Same as previous comment.

hevp commented on source/pages/basics/basic_usage.rst:
```diff
@@ -55,62 +62,35 @@ The landing page of a deposit is created after completion of the online deposit
+To download a single file of the deposit use the ``Download`` button in the files overview. A file can be *online* or *offline*. If a file is *online* it can be downloaded immediately. If a file is *offline* it needs to be staged before it is available for download. To stage a file, login to the data repository and click the ``Request`` button.
 
-You can download all files and optionally the metadata using the 'Download as' dropdown button and choosing 'ZIP'. If all files are online, a ZIP archive file will be offered for download. You can also get a BagIt format archive file.
+To download the entire deposit click the ``Download as`` button in the top right corner of the page. The deposit can be downloaded as ZIP archive or as a BagIt. If not all files are *online* you will receive an error message. To resolve this please stage all files before retrying the operation.
```
'BagIt' => 'BagIt archive file' 

hevp commented on source/pages/basics/downloads.rst:
```diff
@@ -1,76 +1,106 @@
+The recommended method to download multiple deposits or entire collections is the download script. The download script is a tool that automatically downloads all objects and files in your basket or favourites. To download objects using the download script is a three step process:
 
-To download several files at the same time, your can add objects or files to your basket. This can be done by clicking on the 'Add to basket' button in the top-right corner, or by selecting individual files in the files table and then clicking on the basket icon button directly above the table.
+1. install and configure the script;
```
This is good. Let's add another step called 'Add an API token' (as is described later in the section).

hevp reviewed (changes requested):
Looking good, some small changes requested.

jorikvankemenade committed:
Apply review comments to basic usage section

jorikvankemenade committed:
Add API token retrieval to download instructions

jorikvankemenade commented:
@hevp I have rebased on the latest master and applied your review comments (last two commits, 879f4e2 and dab83de). If you agree could you please merge and close? :)

jorikvankemenade committed:
Fix appendix

Fix links in the appendix (remove deadlinks) and convert to rst so that
it is visible in the index.

jorikvankemenade committed:
Remove duplicte text from basic usage section

Also fix warnings from the compiler

jorikvankemenade committed:
Remove unused links from deposit documentation

jorikvankemenade commented:
Pushed three commits to get rid of some warnings from the build step, I have 5 warnings left but those are not caused by these changes:

```
WARNING: Python 2 is deprecated with sphinx_rtd_theme, update to Python 3
WARNING: Sphinx 1.x is deprecated with sphinx_rtd_theme, update to Sphinx 2.x or greater
WARNING: 'html4_writer' is deprecated with sphinx_rtd_theme
...
WARNING: unsupported theme option 'relbarbgcolor' given
WARNING: unsupported theme option 'rightsidebar' given
...
build succeeded, 5 warnings.

The HTML pages are in target.
~/projects/sdrdocs
```

hevp committed:
Merge branch 'master' into doc_review
